ci_framework/roles/cifmw_cephadm/tasks/pools.yml

```diff
@@ -34,7 +35,8 @@
       loop: "{{ cifmw_cephadm_pools | default([]) }}"
       changed_when: false
 
-    - name: Enable application on Ceph pools
+    - name: Enable application on Ceph RBD pools
+      when: item.application == 'rbd'
```

Comment:
+1 on this: we don't need to enable cephfs application because it's implicit for pools generated via `ceph fs volume` cmd. Good to have the 'rbd' condition here.